### DISCUSSION TITLE
Has anyone experienced next build stalling in bitbucket pipelines before?

### DISCUSSION THREAD

**Discussion by conorshaw** [vercel/next.js#38912] [Help]
My build is stalling during webpack compiling during next build when building a docker image in bitbucket pipeline. Build has no errors and eventually the docker container runs out of memory. My project has 0 errors and the build runs successfully locally and also on GitHub Actions but fails in bitbucket. Has anyone experienced this issue before?

I'm using the latest version of next alongside a copy of the Dockerfile from the `with-docker` example https://github.com/vercel/next.js/tree/canary/examples/with-docker

Error:
<img width="451" alt="Screenshot 2022-07-22 at 12 55 16" src="https://user-images.githubusercontent.com/22867304/180434137-2594269e-f6a5-4f68-9749-5c9bb14f0f42.png">

Bitbucket Pipeline:
<img width="374" alt="Screenshot 2022-07-22 at 12 58 13" src="https://user-images.githubusercontent.com/22867304/180434620-fa8ba5b3-8fc2-4628-a49e-94fb50df647a.png">

**Comment by thomasverleye** [comment]
@conorshaw getting pretty desperate here: did you end up fixing the issue and how did you do that?

**Comment by felixizraitel** [comment]
Same thing here. Sometimes works sometimes doesn't in bitbucket. adding this flag seemed to help but still sometimes stalls and fails:

export DOCKER_BUILDKIT=1

**Comment by cmaring** [comment]
Facing the same issue. But I'm not using Docker. I have a bash script that run the npm build command and then upload the build files to a server.
